prow/plugins/label/label.go

```diff
@@ -70,6 +84,40 @@ type githubClient interface {
 	BotName() string
 }
 
+type AliasInterface interface {
+	hasStatusPrivilege(login string) bool
+}
+
+type Alias struct{}
+
+func (Alias) hasStatusPrivilege(login string) bool {
+	resp, err := http.Get(OWNERS_URL)
+
+	if err != nil {
+		fmt.Printf("Failed to get aliases")
+		return false
+	}
+
+	type aliasData struct {
+		// Contains the mapping between aliases and lists of members.
+		AliasMap map[string][]string `json:"aliases"`
+	}
+
+	var data aliasData
+	responseData, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		fmt.Printf("Failed to parse yaml")
+		return false
+	}
+
+	yaml.Unmarshal(responseData, &data)
+	leads, ok := data.AliasMap[sigKey]
+	if ok && sets.NewString(leads...).Has(login) {
```

Comment:
If you're just checking for membership use a `map[string]bool`, no need for a set package.

Reply:
@marun you and @spxtr seem to have differing opinions here.  I'm not super particular one way or another.